core/src/test/scala/scala/pickling/registry/SwitchRuntimeRegistryInit.scala

```diff
@@ -0,0 +1,46 @@
+package scala.pickling.registry
+
+import org.scalatest.FunSuite
+import scala.pickling._
+import scala.pickling.internal.HybridRuntime
+import scala.pickling.internal.DefaultRuntime
+import scala.pickling.json.JsonFormats
+import scala.pickling.pickler.AllPicklers
+
+object Protocol extends {
+  val oldRuntime = internal.currentRuntime
+  val currentRuntime = new HybridRuntime
+  val onlyLookup = internal.replaceRuntime(currentRuntime)
+} with Ops with AllPicklers with JsonFormats
+
+class SwitchRuntimeRegistryInit extends FunSuite {
```

Comment:
I have concerns with this test in the presence of threads being flaky.   

Can we move this to it own sub-module (not core)?   it's a good test, but we should make sure it "owns" all threads to avoid flakyness.